pkgs/by-name/si/signal-desktop/signal-desktop-aarch64.nix

```
@@ -1,8 +1,16 @@
 { callPackage }:
 callPackage ./generic.nix { } rec {
   pname = "signal-desktop";
+  version = "7.43.0-1";
+
   dir = "Signal";
-  version = "7.36.0";
-  url = "https://github.com/0mniteck/Signal-Desktop-Mobian/raw/${version}/builds/release/signal-desktop_${version}_arm64.deb";
-  hash = "sha256-nmAqFDw35pdZg5tiq9MUlqXnbRLRkSOX9SWhccnE2Xw=";
+  libdir = "usr/lib64/${pname}";
+  bindir = "usr/bin";
+  extractPkg = ''
+    mkdir -p $out
+    rpm2cpio $downloadedFile | cpio -imvdD"$out"
```

`libarchive` actually understands `rpm` files so we don't need `rpm`, you can replace this with:

```suggestion
    bsdtar -xf "$downloadedFile" -C "$out"
```
It looks like nixpkgs has a package called `rpmextract` which is designed for this use case, but it still uses `rpm2cpio` under the hood which is [obsolete](https://man7.org/linux/man-pages/man8/rpm2cpio.8.html) as per the man page.